<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettingsFifteen</class>
 <widget class="QDialog" name="DialogSettingsFifteen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings - 15 Puzzle</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>320</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="ImageWidget" name="imageWidget">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="comboBoxGameType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Swap Mode</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/swap</normaloff>:/icons/swap</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slide Mode</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/slide</normaloff>:/icons/slide</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSpecifiedPosition">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Blank position</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonBlankRandom">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Random</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonBlankSpecified">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Specified</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Partition</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBoxSplitY">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="hSliderSplitX">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="hSliderSplitY">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBoxSplitX">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QFrame</extends>
   <header>ImageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxGameType</tabstop>
  <tabstop>radioButtonBlankRandom</tabstop>
  <tabstop>radioButtonBlankSpecified</tabstop>
  <tabstop>hSliderSplitX</tabstop>
  <tabstop>spinBoxSplitX</tabstop>
  <tabstop>hSliderSplitY</tabstop>
  <tabstop>spinBoxSplitY</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettingsFifteen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettingsFifteen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSplitX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxSplitY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSplitX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSliderSplitX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSplitY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hSliderSplitY</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogSettingsFifteen</receiver>
   <slot>udpateGrid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSliderSplitY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogSettingsFifteen</receiver>
   <slot>udpateGrid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>udpateGrid()</slot>
 </slots>
</ui>
